fgdc_config.rb

```diff
@@ -19,52 +21,52 @@
 
 # CHO Required
 to_field 'cho_identifier', generate_fgdc_id
-to_field 'cho_title', extract_fgdc('/*/idinfo/citation/citeinfo/title')
-to_field 'cho_title', extract_fgdc('/*/idinfo/citation/citeinfo/edition')
+to_field 'cho_title', extract_xml('/*/idinfo/citation/citeinfo/title', NS)
```

Comment:
Why aren't you using `extract_fgcd` ? https://github.com/sul-dlss/traject_plus/blob/master/lib/traject_plus/macros/fgdc.rb#L10

Reply:
Seems like a silly macro, but OK. Fixing.